packages/forms/src/UIForm/fieldsets/Array/Array.component.js

```diff
@@ -149,7 +149,9 @@ if (process.env.NODE_ENV !== 'production') {
 		onFinish: PropTypes.func.isRequired,
 		schema: PropTypes.object.isRequired,
 		templates: PropTypes.object.isRequired,
-		value: PropTypes.arrayOf(PropTypes.object).isRequired,
+		value: PropTypes.arrayOf(
```

Comment:
maybe boolean to ?

Reply:
Oh thanks

Reply:
I was wondering if it's not one case to use `any` as it could really be an array of anything, even an array of array potentially...

Reply:
Deal, changed, pushed